packages/components/psammead-brand/src/index.jsx

```diff
@@ -174,6 +174,7 @@ const Brand = props => {
     borderBottom,
     backgroundColour,
     logoColour,
+    ...rest
```

Comment:
Is it `...rest` the same as `...props`? If so, can we keep the same naming across files?

Reply:
Ok, the reason for naming it `...rest` here is because `props` is already used.

Reply:
I'd have called it `additionalProps` or something, but that's up to you.